src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -473,9 +474,19 @@ internal void PopulateAutoCompleteCandidates()
             if (IsDisplayingMLRecommendation)
             {
                 DisplayMachineLearningResults();
+                //Tracking Analytics when raising Node Autocomplete with the Recommended Nodes option selected (Machine Learning)
+                Analytics.TrackEvent(
+                    Actions.SelectMachineLearning,
+                    Categories.NodeAutoCompleteOperations,
+                    "DisplayingMLRecommendations");
```

Comment:
We are missing the tracking of this flip mode from preferences as well, since this is related to preferences, I would maybe track it under preference category

Reply:
Added tracking for when the Autocomplete functionality is changed to ML Recommendation or ObjectType.
See the next commit: https://github.com/DynamoDS/Dynamo/pull/13581/commits/0655f6d242fa449dc828a75d8634567d3b5737f4